apiserver/facades/controller/caasfirewaller/firewaller_test.go

```diff
@@ -141,27 +133,22 @@ func (s *firewallerSidecarSuite) TestWatchOpenedPorts(c *gc.C) {
 }
 
 func (s *firewallerSidecarSuite) TestGetApplicationOpenedPorts(c *gc.C) {
-	ctrl := s.getCtrl(c)
-	defer ctrl.Finish()
-
-	gomock.InOrder(
-		s.appPortRanges.EXPECT().ByEndpoint().Return(network.GroupedPortRanges{
-			"": []network.PortRange{
-				{
-					FromPort: 80,
-					ToPort:   80,
-					Protocol: "tcp",
-				},
+	s.st.application.appPortRanges = network.GroupedPortRanges{
+		"": []network.PortRange{
+			{
+				FromPort: 80,
+				ToPort:   80,
+				Protocol: "tcp",
 			},
-			"endport-1": []network.PortRange{
-				{
-					FromPort: 8888,
-					ToPort:   8888,
-					Protocol: "tcp",
-				},
+		},
+		"endport-1": []network.PortRange{
+			{
+				FromPort: 8888,
+				ToPort:   8888,
+				Protocol: "tcp",
 			},
-		}),
-	)
+		},
+	}
 
 	results, err := s.facade.GetApplicationOpenedPorts(params.Entity{
```

Comment:
Is "GetApplicationOpenedPorts" still the correct name for this? 
We already changed the uniter agent api,  this one should just be "GetOpenedPorts" right?